### PR TITLE
{lyn13481} Update the Asset Cache Server web page with UX

### PR DESCRIPTION
**Pull request by jackalbe**
Added the UX dialog and how to use it
Updated the configuration JSON fields

Signed-off-by: Allen Jackson <23512001+jackalbe@users.noreply.github.com>

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/assets/asset-processor/asset-cache-server.md [hunk 1]
```diff
@@ -22,77 +22,151 @@ The next sections assume the root folder to be `T:/o3de` but it can be in any fo
...
+
+The `Save Changes` and `Discard` buttons become enabled when a change to the settings has been detected. The `Save Changes` commits the changes to a settings file. The `Discard` button reverts the panel's values.
+
+The settings file is written to the `{project_folder}/Registry/asset_cache_server_settings.setreg` file. 
+
+If the Remote Folder is set to an invalid folder location, then an error dialog will show up and no settings will be written.
+
+# Asset Cache Server: Settings Details
+
+The settings for the Asset Cache Server are held in `.setreg` files inside the project. For example the `o3de/AutomatedTesting/Registry/editorpreferences.setreg` stores the settings for the Editor.
```
Comment (AMZN-stankowi) [inline content/docs/user-guide/assets/asset-processor/asset-cache-server.md:76]: You might want to clarify that those are just the settings for the Editor for the Automated Testing project, and not other projects.

**Review by AMZN-stankowi — APPROVED** [review]

File: content/docs/user-guide/assets/asset-processor/asset-cache-server.md [hunk 2]
```diff
@@ -22,77 +22,151 @@ The next sections assume the root folder to be `T:/o3de` but it can be in any fo
...
+### Manage Shared Cache Patterns
+
+The table shows the active asset patterns that will be cached when the asset is processed on the server. The asset pattern row contains:
+ 1. a check box to enable or disable the pattern
+ 2. a text box to store the pattern's name
+ 3. a combo box how to use the pattern to match source assets (Wildcard or RegEx)
+ 4. a text box to enter the pattern to match source assets
+ 5. a button to remove the pattern
+ 
+The enable flag allows a user to toggle the row to disabled to force the local client Asset Processor to process the matching assets locally. Most of the time the client users will have all patterns enabled.
```
Comment (micronAMZN) [inline content/docs/user-guide/assets/asset-processor/asset-cache-server.md:58]: Minor suggestion. Saying the local server processes the assets locally seems redundant.

```suggestion
The enable flag allows a user to toggle the row to disabled to force the local client Asset Processor to process the matching assets. Most of the time the client users will have all patterns enabled.
```

**Review by micronAMZN — APPROVED** [review]
Approved with minor suggestions.

File: content/docs/user-guide/assets/asset-processor/asset-cache-server.md [hunk 3]
```diff
@@ -22,77 +22,151 @@ The next sections assume the root folder to be `T:/o3de` but it can be in any fo
...
 
-## Run Asset Processor in ACS mode as a server
+## Using the Shared Cache Tab
+
+The Asset Processor has a tab that manages the settings for the Shared Cache (aka Asset Cache Server).
+
+The panel is broken up into three steps:
+ 1. Choose a mode via "Set shared cache mode"
+ 2. Select the transfer directory via "Select a remote folder"
+ 3. Select the asset patterns that will be cached via "Manage shared cache patterns"
```
[the remark this answers is not in the document]
Comment (jackalbe) [reply content/docs/user-guide/assets/asset-processor/asset-cache-server.md:32]: Talked with Josh. We are going to keep the "Steps" for now.

**Pull request merged**